blog/2024/04-28-jellyfin-release-10.9.0/index.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Jellyfin 10.9.0
+description: We are pleased to announce the latest stable release of Jellyfin, version 10.9.0!
+authors: joshuaboniface
+date: 2024-04-28
+slug: jellyfin-release-10.9.0
```

Comment:
```suggestion
slug: jellyfin-10.9.0
```

please keep it consistent with the previous posts

Reply:
Worded badly - basically I need a way to clearly differentiate release blog posts from everything else. I went with this new format both to differentiate for that purpose, and ensure that. They are releases, so having release in the slug seems like a trivial change to me.

Reply:
Ok so going forward the server releases will use this format then?

Reply:
Correct, the bot looks for this specific slug/branch format to auto-link everything.